dataland-frontend/tests/e2e/fixtures/sfdr/SfdrDataFixtures.ts

```diff
@@ -58,80 +51,109 @@ export function generateSfdrData(nullProbability = DEFAULT_PROBABILITY): SfdrDat
+        policyAgainstForcedLabour: dataGenerator.randomBaseDataPoint(generateYesNo()),
+        policyAgainstDiscriminationInTheWorkplace: dataGenerator.randomBaseDataPoint(generateYesNo()),
+        iso14001Certificate: dataGenerator.randomBaseDataPoint(generateYesNo()),
+        policyAgainstBriberyAndCorruption: dataGenerator.randomBaseDataPoint(generateYesNo()),
         fairBusinessMarketingAdvertisingPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
         technologiesExpertiseTransferPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
         fairCompetitionPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
         violationOfTaxRulesAndRegulation: dataGenerator.randomExtendedDataPoint(generateYesNo()),
-        unGlobalCompactPrinciplesCompliancePolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
-        oecdGuidelinesForMultinationalEnterprisesPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
+        unGlobalCompactPrinciplesCompliancePolicy: dataGenerator.randomBaseDataPoint(generateYesNo()),
+        oecdGuidelinesForMultinationalEnterprisesGrievanceHandling: dataGenerator.randomExtendedDataPoint(
+          generateYesNo(),
+        ),
         averageGrossHourlyEarningsMaleEmployees: dataGenerator.randomCurrencyDataPoint(),
         averageGrossHourlyEarningsFemaleEmployees: dataGenerator.randomCurrencyDataPoint(),
         femaleBoardMembers: dataGenerator.randomExtendedDataPoint(generateFloat()),
         maleBoardMembers: dataGenerator.randomExtendedDataPoint(generateFloat()),
         controversialWeaponsExposure: dataGenerator.randomExtendedDataPoint(generateYesNo()),
-        workplaceAccidentPreventionPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
-        rateOfAccidents: dataGenerator.randomExtendedDataPoint(generateFloat()),
+        workplaceAccidentPreventionPolicy: dataGenerator.randomBaseDataPoint(generateYesNo()),
+        rateOfAccidentsInPercent: dataGenerator.randomExtendedDataPoint(generatePercentageValue()),
         workdaysLostInDays: dataGenerator.randomExtendedDataPoint(generateFloat()),
-        supplierCodeOfConduct: dataGenerator.randomExtendedDataPoint(generateYesNo()),
+        supplierCodeOfConduct: dataGenerator.randomBaseDataPoint(generateYesNo()),
         grievanceHandlingMechanism: dataGenerator.randomExtendedDataPoint(generateYesNo()),
-        whistleblowerProtectionPolicy: dataGenerator.randomExtendedDataPoint(generateYesNo()),
+        whistleblowerProtectionPolicy: dataGenerator.randomBaseDataPoint(generateYesNo()),
         reportedIncidentsOfDiscrimination: dataGenerator.randomExtendedDataPoint(generateFloat()),
-        sanctionsIncidentsOfDiscrimination: dataGenerator.randomExtendedDataPoint(generateFloat()),
-        ceoToEmployeePayGap: dataGenerator.randomExtendedDataPoint(generateFloat()),
+        sanctionedIncidentsOfDiscrimination: dataGenerator.randomExtendedDataPoint(generateFloat()),
```

Comment:
This an a lot of other fields should be generated as Integers.

Reply:
Done